### ISSUE TITLE
Windows Defender

### ISSUE THREAD

**Issue by ghost** [buckyroberts/Python-Reverse-Shell#46]
Windows Defender or firewall is stoping to create that type of file...
Any solution???

**Comment by MuhammadAli947** [comment]
Yes basically in turtle Roberts is using win32  and executables to create executable for client. But if we use simple a pyinstaller to create --onfile it will not create any problem you can use it with subprocess or using the terminal directly as well.
**pyinstaller --onefile client.py**
